### PR TITLE
New icon for address switcher

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx [hunk 1]
```diff
@@ -23,12 +23,17 @@ const AddressDisplay: React.FC<Props> = ({ bagId, headingSize = "h2", isHeader =
   const { data } = useBAG(bagId)
   const { data: otherAddresses } = useOtherAddressesByBagId(bagId)
 
+  const isCurrentAddress = (address: any) => address.adres.trim() === data?.results[0].adres.trim()
```
Comment (gilleswittenberg) [inline src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx:26]: Does `const isCurrentAddress = (address: { adres: string }) => address.adres.trim() === data?.results[0].adres.trim()` work?

File: public/icons/chevron_up_down.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                <g id="Path" transform="translate(0.000000, 17.777778)">
+                    <polygon points="11.3777778 14.049341 0 2.67156323 2.06935162 0.609523812 11.3777778 9.9106378 20.6862039 0.609523812 22.7555556 2.67156323"></polygon>
+                </g>
+                <g id="Path" transform="translate(11.377778, 7.111111) rotate(-180.000000) translate(-11.377778, -7.111111) translate(0.000000, 0.000000)">
+                    <polygon points="11.3777778 14.049341 0 2.67156323 2.06935162 0.609523812 11.3777778 9.9106378 20.6862039 0.609523812 22.7555556 2.67156323"></polygon>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (gilleswittenberg) [inline public/icons/chevron_up_down.svg:15]: Could be minimized https://www.svgminify.com/

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx:26]: Yes, it does :-)
Any doubts?

File: public/icons/chevron_up_down.svg (hunk 2 above)
Comment (remcohoff) [reply public/icons/chevron_up_down.svg]: Good point, done

File: src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx (hunk 1 above)
Comment (gilleswittenberg) [reply src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx:26]: Not if it works :)

**Pull request merged**